content/knowledge/scalinghistory.md

```diff
@@ -0,0 +1,144 @@
+# A Brief History of Decentralised Scaling
+
+This article covers how the scaling solutions for decentralized systems in the crypto world have evolved with regards to the consensus algorithms that have been used, what their problems are and how Flare tries to address them.
+It also addresses how various blockchains fare in the realm of the [blockchain trilemma](https://vitalik.ca/general/2021/04/07/sharding.html) which roughly states that it is hard to achieve more than 2 ouf 3 from security, scalability and decentralization.
+
+The concepts at the foundation of blockchain technology, such as cryptography (for data encryption) and peer-to-peer networks (that handle distributed information being exchanged between the participants) were around before the 2000s.
+However, there was no concept of a truly digital currency due to the [double spending problem](https://en.wikipedia.org/wiki/Double-spending).
+For example, say Brian has $100 worth of digital value, and Brian sends $100 to both Alice and Bob at the same time.
+There used to be no fully robust way to determine which one of the two transactions is the valid one.
```

Comment:
```suggestion
```

My previous suggestion involved not only adding a link, but also avoiding examples like this one, since I feel like they are not making the document clearer IMO.

Reply:
Just to understand, is the example not representative of what an actual double spending problem may look like?
 
My main reason to add it was so that a beginner reader will understand the basic issue without actually having to go through the given external link. But if this example isn't making the idea clearer then I am happy to remove it. :slightly_smiling_face: